docs/GettingStartedWindows.md

```diff
@@ -66,8 +78,9 @@ git clone git://github.com/compnerd/swift-build-examples /SourceCache/swift-buil
 
 2. Setup Common Build Parameter Variables
 
+If you've already set the ```SDKROOT``` parameter variable as instructed above then simply add the following:
```

Comment:
Sorry, can’t read this properly on a phone.  The reason for the duplication is that the previous steps download/install the module map files, and are a one time thing.  This section is for reach time that you run the shell.  I think that making that easy by being copy paste friendly is valuable, and resetting the variable is fine.